legacy/src/api/payment-requests/payment-requests.md

```diff
@@ -103,12 +103,13 @@ version (documented on this page) and the "legacy" version (documented at
 
 {% h4 Optional Fields %}
 
-|     Field      |  Type  |                                      Description                                      |
-| -------------- | ------ | ------------------------------------------------------------------------------------- |
-| bitcoinAddress | String | ★  Address to send Bitcoin, when the "assetType" is `bitcoin.*`                       |
-| cennzAddress   | String | ★  Address to send CPay, when the "assetType" is `cennznet.*`                         |
-| wavesAddress   | String | ★  Waves address to send Zap tokens, when the "assetType" is `zap.*`                  |
-| productCodes   | Array  | Supported product codes for the payment request, when the "assetType" is `epay.nzd.*` |
+|     Field      |  Type  |                                        Description                                         |
+| -------------- | ------ | ------------------------------------------------------------------------------------------ |
+| bitcoinAddress | String | ★  Address to send Bitcoin, when the "assetType" is `bitcoin.*`                            |
+| cennzAddress   | String | ★  Address to send CPay, when the "assetType" is `cennznet.*`                              |
+| wavesAddress   | String | ★  Waves address to send Zap tokens, when the "assetType" is `zap.*`                       |
+| productCodes   | Array  | Supported product codes for the payment request, when the "assetType" is `epay.nzd.*`      |
+| collections    | Array  | Supported collections for the payment request, when the "assetType" is `centrapay.token.*` |
```

Comment:
Why is this an array of objects rather than an array of `collectionId`s?

Reply:
I agree with @Ray2126, this should be an array of collectionIds. I also think that the field should be renamed to collectionIds.